website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -82,7 +76,7 @@ Install Consul on Kubernetes by using the CLI to apply `values.yaml` to each clu
       ```
 
       ```shell-session
-    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.0" --values values.yaml --kube-context $CLUSTER1_CONTEXT
+    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --kube-context $CLUSTER1_CONTEXT
```

Comment:
nit: It is a little confusing (possibly wrong somewhere?) that we do not set `global.datacenter` for dc1 in the `values.yaml` OR on command line, and then use `--set global.datacenter=dc2` on command line for the dc2 example. I think people might miss this detail, does it make sense to call it out or be explicit in both values.yaml or helm commands?